lib_nbgl/include/nbgl_use_case.h

```diff
@@ -157,6 +175,11 @@ void nbgl_useCaseSettings(const char                *settingsTitle,
                           nbgl_callback_t            quitCallback,
                           nbgl_navCallback_t         navCallback,
                           nbgl_layoutTouchCallback_t controlsCallback);
+void nbgl_useCaseSettingsV2(const char                   *title,
```

Comment:
Don't you have a more sexy name :wink: ?
`nbgl_useCaseGenericSettings`? Is it better?

Reply:
`nbgl_useCaseGenericSettings` adopted!